### DISCUSSION TITLE
is there a GUI?

### DISCUSSION THREAD

**Discussion by ghost** [AviSynth/AviSynthPlus#355] [Q&A] [answered]
are there any active GUI's for avisynth+?

**Comment by Purfview** [comment, marked as answer]
There is:
https://github.com/gispos/AvsPmod
https://forum.doom9.org/showthread.php?t=175823

**Comment by qyot27** [comment]
Really, define 'GUI'.  Because that's conditional on what somebody is expecting of a GUI.  A script editor with a preview?  Yeah, AvsPmod.  However, the problem there is the extreme non-portability of the fact that it hasn't been updated to Python 3 and that any of the cross-platform fixes for the AvxSynth bringup that existed in the original repo have been nerfed through code churn in the years since - and fixing them again for AviSynth+ basically means using a Python version that all those other OSes haven't deprecated and stopped shipping years ago.
